views/policies.py

```diff
@@ -0,0 +1,183 @@
+from common.ui_utils import ui_utils
+from common.navigate import navigate
+from common.db import db
+
+class policies():
+    web_session = None
+    web_driver = None
+    ui_utils = None
+
+    EVENT_TYPES = ('deployments', 'datasources', 'jdbc')
+
+    def __init__(self, web_session):
+        self.web_session = web_session
+        self.web_driver = web_session.web_driver
+        self.ui_utils = ui_utils(self.web_session)
+
+    def add_middleware_control_policy(self, policy_name, product_name, delete_policy=False, assignment_events=None):
+        self.web_session.logger.info('Add Policy: "{}", Product Name: "{}"'.format(policy_name, product_name))
+
+        if db(self.web_session).is_policy_present(policy_name):
+            if delete_policy:
+                if not self.delete_middleware_control_policy(policy_name):
+                    return False
+            else:
+                self.web_session.logger.info('Policy "{}" already exists.'.format(policy_name))
+                return True
+
+        self.navigate_to_middleware_server_control_policies()
+
+        # Add the policy
+
+        self.web_driver.find_element_by_xpath('//*[@title="Configuration"]').click()
+        self.ui_utils.sleep(2)
+        el = self.web_driver.find_element_by_xpath('//*[@title="Add a New Middleware Server Control Policy"]')
+        #self.ui_utils.wait_until_element_displayed(mw_el, 10)
+        el.click()
+        assert self.ui_utils.waitForTextOnPage("Notes", 15)
+
+        # Fill out form
+        self.web_driver.find_element_by_id("description").send_keys(policy_name)
+        self.web_driver.find_element_by_xpath("//span[contains(.,'Choose')]").click()
+        el = self.web_driver.find_element_by_xpath("//span[contains(.,'Field')]")
+        assert self.ui_utils.wait_until_element_displayed(el, 10)
+        el.click()
+        self.ui_utils.sleep(2)
+        self.web_driver.find_elements_by_xpath(".//*[contains(text(), 'Choose') and not (ancestor::*[contains( @ style, 'display: none')])]")[3].click()
+
+        el = self.web_driver.find_element_by_xpath("//*[contains(text(),'Middleware Server : Product')]")
+        self.ui_utils.wait_until_element_displayed(el, 10)
+        el.click()
+        self.ui_utils.sleep(2)
+        el = self.web_driver.find_element_by_xpath('//*[@id="exp_atom_editor_div"]/fieldset/div[4]/button/span[1]')
+        assert self.ui_utils.wait_until_element_displayed(el, 10)
+        el.click()
+        el = self.web_driver.find_element_by_xpath("//*[contains(text(),'INCLUDES')]")
+        assert self.ui_utils.wait_until_element_displayed(el, 10)
+        el.click()
+        self.ui_utils.sleep(2)
+        el = self.web_driver.find_element_by_id('chosen_value')
+        assert self.ui_utils.wait_until_element_displayed(el, 10)
+        el.send_keys(product_name)
+
+        # Click on Check box
+        el = self.web_driver.find_element_by_class_name('fa-check').click()
```

Comment:
@mattmahoneyrh 
Fa-check is font-awesome class.

What happens if there are more than one element using this class?